core/internal/features_test.go

```diff
@@ -527,7 +528,7 @@ func TestIntegration_OCR(t *testing.T) {
 
 	// Note it's plausible these ports could be occupied on a CI machine.
 	// May need a port randomize + retry approach if we observe collisions.
-	appBootstrap, bootstrapPeerID, _, _, bootstrapCfg, cleanup := setupNode(t, owner, 19999, "bootstrap", b)
+	appBootstrap, bootstrapPeerID, _, _, _, cleanup := setupNode(t, owner, 19999, "bootstrap", b)
 	defer cleanup()
```

Comment:
I added this while debugging a separate issue (it made an error in stopping more obvious).

I left the waitgroup because it speeds up shutdown by closing all apps in parallel rather than sequentially, making the test faster.